### ISSUE TITLE
[REQUEST] Syslogd, FAZ, FCTLD

### ISSUE THREAD

**Issue by rhumbert54** [AsBuiltReport/AsBuiltReport.Fortinet.FortiGate#46]
### Description

Hi,

It could be interesting to add settings of logging container about the FortiGate.

Regards,
Renaud

### Additional Context

Fourth could exist, if state disable or null, then discard.
show log syslogd setting
show log syslogd2 setting
show log syslogd3 setting
show log syslogd4 setting

Three could exist, if state disable or null, then discard.
show log fortianalyzer setting
show log fortianalyzer2 setting

If no value, then discard
show log fortianalyzer-cloud setting
show log fortiguard setting

### Before submitting

- [X] I have read [the documentation](https://www.asbuiltreport.com/user-guide/new-asbuiltconfig), and referred to the [known issues](https://www.asbuiltreport.com/user-guide/known-issues/) before submitting this change request.
- [X] I have checked for previously opened & closed [issues](https://github.com/AsBuiltReport/AsBuiltReport.Fortinet.FortiGate/issues) before submitting this change request.

**Comment by alagoutte** [comment]
Yes ! Good idea ! But need to add get function before on PowerFGT module !
